tests/test_web/test_api_v1.py

```diff
@@ -225,22 +227,21 @@ def test_get_unique_bundles(bundles, exp_bundles, app):
 @pytest.mark.parametrize(
     ('logs_content', 'expired', 'finalized', 'expected'),
     (
-        ('foobar', False, False, {'status': 200, 'mimetype': 'text/plain', 'data': 'foobar'}),
-        ('foobar', True, False, {'status': 200, 'mimetype': 'text/plain', 'data': 'foobar'}),
-        ('', False, False, {'status': 200, 'mimetype': 'text/plain', 'data': ''}),
-        ('', True, False, {'status': 200, 'mimetype': 'text/plain', 'data': ''}),
-        (None, False, False, {'status': 200, 'mimetype': 'text/plain', 'data': ''}),
+        ('foobar', False, True, {'status': 200, 'mimetype': 'text/plain', 'data': 'foobar'}),
+        ('foobar', True, True, {'status': 200, 'mimetype': 'text/plain', 'data': 'foobar'}),
+        ('', False, True, {'status': 200, 'mimetype': 'text/plain', 'data': ''}),
+        ('', True, True, {'status': 200, 'mimetype': 'text/plain', 'data': ''}),
         (
             None,
             True,
-            False,
+            True,
```

Comment:
Is it necessary to keep `finalized` column/parameter since it is always `True`?

On the other hand, is it possible to test `get_build_logs` on incomplete request or logs not found? (Or creating JIra issue if it not in scope)

Reply:
:grey_question:

Reply:
I already added the test for incomplete request in this PR :)

Reply:
So I believe we don't have to file a separate Jira for it anymore